Sources/Core/Schema.swift

```diff
@@ -257,6 +261,19 @@ extension Schema {
             return [ref.url]
         }
     }
+
+    func objectRoots() -> Set<SchemaObjectRoot> {
```

Comment:
Does this handle the`oneOf` / ADT case? I image we want to support inline types there as well. It should just be adding another case to this method

Reply:
It didn't, but I've added support for that.